packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -156,7 +156,7 @@ export function computeDefaults<
     ) as T[];
   } else if (ONE_OF_KEY in schema) {
     schema = schema.oneOf![
-      getMatchingOption<T, S, F>(
+      getFirstMatchingOption<T, S, F>(
```

Comment:
@nickgros I'm wondering if, because of value defaulting, we actually want to use `getClosestMatchingOption()` here instead?? Is the slower performance for the best choice worth it? I'm on the fence but leaning towards best vs fast